src/Jab/build/Jab.props

```diff
@@ -0,0 +1,5 @@
+﻿<Project>
+    <PropertyGroup>
+        <JabAttributesReferenced>false</JabAttributesReferenced>
```

Comment:
What is this property used for?

Reply:
It is used to control the compiler constant on `Jab.targets` file

```xml
    <Target Name="_JabReferences" DependsOnTargets="ResolvePackageAssets">
        <PropertyGroup>
            <JabAttributesReferenced
                Condition="@(PackageDependencies->WithMetadataValue('Identity', 'Jab.Attributes')->Count()) != 0">true</JabAttributesReferenced>
            <DefineConstants Condition="$(JabAttributesReferenced)">$(DefineConstants);JAB_ATTRIBUTES_REFERENCED</DefineConstants>
        </PropertyGroup>
    </Target>
```

Reply:
Is it required to be pre-initialized in a props file?


Can we 

```
    <Target Name="_JabReferences" DependsOnTargets="ResolvePackageAssets">
        <PropertyGroup>
            <_JabAttributesReferenced
                Condition="@(PackageDependencies->WithMetadataValue('Identity', 'Jab.Attributes')->Count()) != 0">true</JabAttributesReferenced>
            <DefineConstants Condition="'$(_JabAttributesReferenced)' == 'True'">$(DefineConstants);JAB_ATTRIBUTES_REFERENCED</DefineConstants>
        </PropertyGroup>
    </Target>
```

Reply:
No, it's not. I'll change as suggested

Reply:
Done